src/Components/Card/Card.js

```diff
@@ -7,7 +7,7 @@ import MovieInfo from '../MovieInfo/MovieInfo'
 function Card({movieTitle, movieId, moviePoster, movieRating, releaseDate, backdrop}) {
 
   return (
-    <div className = "movie-card" >
+    <div className = "movie-card" placeholder="mov-card">
```

Comment:
What's the placeholder doing for you here?

Reply:
If you're using it for a test query, I would opt to query the movie's title instead